src/benchmarkstt/csv.py

```diff
@@ -166,15 +161,15 @@ def next_field():
             field = []
             mode = MODE_OUTSIDE
 
-        debug = self._debug
-        if debug:
-            # print the color key the different modes
-            current_module = sys.modules[__name__]
-            print('MODES: ', end='')
-            print(' '.join(['\033[1;%d;40m%s\033[0;0m' % (32 + getattr(current_module, name), name[5:])
-                            for name in dir(current_module)
-                            if name.startswith('MODE_')
-                            ]))
+        # debug = True
```

Comment:
It can, but in this specific file case I would prefer to keep it in place as it is some very useful debugging code if we would ever need it, it was commented out for performance and code coverage reasons.
Alternatively I could make a debug method that would attempt to debug or do nothing, this would add a function call, but would be more readable. I would then exclude those lines from code coverage though with `# pragma: nocover`.

Are you okay with either making it available again, but without code coverage checks, or leaving it commented out?

Reply:
A debug method without coverage checks would be cleaner in my opinion.

Reply:
Commit 
17122dc 
I've gone above and beyond and added a unit test as well, trying to minimize the "no coverage" blocks...